install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -152,6 +152,16 @@ $ ansible-playbook [-i /path/to/file] \
   /usr/share/ansible/openshift-ansible/playbooks/openshift-etcd/scaleup.yml
 ----
 
+. If you use the service catalog, you must update its list of etcd servers:
++
+[source,bash]
+----
+$ oc edit ds apiserver -n kube-service-catalog
+----
++
+Add the etcd URL for the new etcd node to the `--etcd-servers` argument. This
```

Comment:
Same suggestion as above about removing "etcd" in front of URL if it makes sense.